includes/virtual-machines-disks-incremental-snapshot.md

```diff
@@ -75,11 +75,13 @@ $incrementalSnapshots
 
 ## CLI
 
-You can create an incremental snapshot with the Azure CLI, you will need the latest version of Azure CLI. The following command will either install or update your existing installation to the latest version:
+You can create an incremental snapshot with the Azure CLI, you will need the latest version of Azure CLI. 
 
+On Windows,the following command will either install or update your existing installation to the latest version:
 ```PowerShell
 Invoke-WebRequest -Uri https://aka.ms/installazurecliwindows -OutFile .\AzureCLI.msi; Start-Process msiexec.exe -Wait -ArgumentList '/I AzureCLI.msi /quiet'
 ```
+On Linux, the CLI installation will vary depending on operating system version.  Please see [Install the Azure CLI](https://docs.microsoft.com/en-us/cli/azure/install-azure-cli) for your particular Linux version.
```

Comment:
If you would remove "Please" that would be appreciated.

Reply:
done!